.babelrc

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I decided to set up babel, so we can write under the same standard both in frontend and backend.